src/porepy/models/solution_strategy.py

```diff
@@ -131,12 +132,49 @@ def __init__(self, params: Optional[dict] = None):
         )
         """Time manager for the simulation."""
 
+        self.restart_options: dict = params.get(
+            "restart_options",
+            {
+                "restart": False,
+                # Boolean flag controlling whether restart is active. Internally
+                # assumed to be False.
+                "pvd_file": None,
+                # Path to pvd file collecting either multiple time steps (generated
```

Comment:
Is this a `Path`?

Reply:
No, you are right. The main motivation to provide `pvd_file` here was to discuss the keywords. The default restart options hold anyhow an inactive state. Thus no keyword will be visited. Nevertheless, to comply with the expected typing, one could also use `Path(".")` here. Would this be better?

Reply:
I think `None` is fine, but the expected type should be mentioned in the documentation.